fix(tutorials): wordpress & lemp stack MTA-1593 (#269)
scaleway/docs-content: tutorials/wordpress-lemp-stack/index.mdx

  - compute
  - instances
dates:
  validation: 2021-12-07
  posted: 2018-11-09
---

## Wordpress with LEMP - Overview

WordPress is a popular, free open-source blogging tool and content management system (CMS) based on PHP and MySQL. WordPress has seen incredible adoption rates, and is a great choice for getting a website up and running quickly. After setup, almost all the administration can be done through the web frontend.

> In this tutorial, we will learn how to install WordPress on a freshly created Ubuntu Bionic Beaver instance with LEMP (Linux + Nginx - pronounced "engine x" + MySQL + PHP). Nginx is a HTTP server that, compared to Apache, uses less resources and delivers pages much faster, especially static files.

Reviewer: ```suggestion
In this tutorial, we will learn how to install WordPress on a freshly created Ubuntu Bionic Beaver instance with LEMP (Linux + Nginx - pronounced "engine x" + MySQL + PHP). Nginx is an HTTP server that, compared to Apache, uses less resources and delivers pages much faster, especially static files.
```

<Message type="requirement">

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
- You have sudo privileges or access to the root user.
- You have [LEMP installed](/tutorials/installation-lemp-ubuntu-bionic/)
- You have a FQDN (Fully Qualified Domain Name) pointing to your Instance’s IP address
- You have [secured your LEMP stack with TLS/SSL](/tutorials/installation-lemp-ubuntu-bionic/#configuring-ssl-with-let's-encrypt)

</Message>

## Installing the stack

> As a tutorial on LEMP is already available, you can follow the procedure at [Installing LEMP on Ubuntu Bioic Beaver.](/tutorials/installation-lemp-ubuntu/)

Reviewer 2: ```suggestion
As a tutorial on LEMP is already available, you can follow the procedure at [Installing LEMP on Ubuntu Bionic Beaver.](/tutorials/installation-lemp-ubuntu/)
```

            root /var/www/wordpress;
            index index.php index.html index.htm;

            server_name blog.example.com;

>             #error_page 404 /404.html;

Reviewer 3: ```suggestion
```
Remove this line as it is commented-out and no required

>             #error_page 500 502 503 504 /50x.html;

Reviewer 3: ```suggestion
```
Remove this line as it is commented-out and not required

            location ~ \.php$ {
                    try_files $uri =404;
                    fastcgi_split_path_info ^(.+\.php)(/.+)$;

>                     #fastcgi_pass unix:/var/run/php7.2-fpm.sock;

Reviewer 3: ```suggestion
```
Remove this line as it is commented-out and not required

  rm /etc/nginx/sites-enabled/default
  ```

  Steps 6 and 7 optimize your Nginx configuration for Wordpress:

6. Open the Nginx configuration file:

  ```
  nano /etc/nginx/nginx.conf
  ```

7. Edit the following to optimize:

    <Message type="tip">

>     Ensure the number of worker processes equals to the number of cores in your Instance

Reviewer: ```suggestion
    Ensure the number of worker processes is equal to the number of cores in your Instance.
```

1. Select your language.

>   You are asked to enter your database information. Before doing so, you need to go back to the terminal to complete the next steps.

Reviewer: ```suggestion
  You are asked to enter your database information. Before doing so, go back to the terminal to complete the next steps.
```
